project/plugins.sbt

```diff
@@ -1,9 +1,5 @@
-libraryDependencies <+= sbtVersion(v => v.split('.') match {
-    case Array("0", "13", _) => "org.scala-sbt" % "scripted-plugin" % v
-    case Array("0", "12", _) => "org.scala-sbt" % "scripted-plugin" % v
-    case _                   => "org.scala-sbt" %% "scripted-plugin" % v
-})
+libraryDependencies <+= sbtVersion(v => "org.scala-sbt" % "scripted-plugin" % v)
 
 addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.5.1")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
```

Comment:
I think it would be better to place these two plugins in `~/.sbt/0.13/plugins/gpg.sbt` as suggested in http://www.scala-sbt.org/sbt-pgp/ .